examples/benchmark/README.md

```diff
@@ -0,0 +1,43 @@
+## Clickstream benchmark tool
+
+We have developed a tool for benchmark tests, which utilizes ECS Fargate to continuously send mock data to Clickstream. The number of ECS Service tasks can be increased or decreased to adjust the RPS (Requests per Second).
+
+### Prerequesisities
+- Make sure you have an AWS account
+- Configure [credential of aws cli][configure-aws-cli]
```

Comment:
Is it a broken link after rendering the markdown?

Reply:
Yes, thank you to find this.
Fixed